.github/workflows/forge.yml

```
@@ -0,0 +1,23 @@
+name: Forge Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  run-ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
```

```suggestion
      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579

      - name: Install Foundry
        uses: onbjerg/foundry-toolchain@ca7e86820a6802e19349b5622e91976c4561c454
```

While this is not a high risk workflow i'd be interested to get your thoughts on pinning actions versions?

---

no thx